spec/io_spec.rb

```diff
@@ -146,4 +146,28 @@
     expect(o).to eq(m)
     expect(o).not_to eq(n)
   end
+
+  it "reads csv file containing a single dense integer matrix" do
```

Comment:
You're writing a plugin, not making additions to the nmatrix gem. These tests are supposed to be in the `spec/plugins/csv` folder. Have you read Will Levine's post entirely? Also, test running only these tests with `rake spec nmatrix_plugins=csv`. No other plugin should run when you do so.

You'll also need to make additions to the `travis.sh` and `travis.yml` files so that the plugin is incorporated in Travis CI.